charts/portal/values.yaml

```diff
@@ -399,8 +399,22 @@ backend:
     frameDocumentTypeIds:
       type0: "CX_FRAME_CONTRACT"
     onboardingServiceProvider:
-      # -- Client-secret for onboardingserviceprovider encryptionKey. Secret-key 'onboardingserviceprovider-encryption-key'.
-      encryptionKey: ""
+      encryptionConfigIndex: 1
+      encryptionConfigs:
+        index0:
+          index: 0
+          cipherMode: "ECB"
+          paddingMode: "PKCS7"
+          # -- Client-secret for onboardingserviceprovider encryptionKey. Secret-key 'onboardingserviceprovider-encryption-key0'.
```

Comment:
Even tough this was already the description beforehand: I think it would be less confusing to write "EncryptionKey for onboardingserviceprovider" than ""Client-secret for onboardingserviceprovider encryptionKey". Same for other occurrences.

Reply:
done